app/javascript/packs/ial2-consent-button.js

```diff
@@ -1,10 +1,9 @@
 function toggleButton() {
-  const continueButton = document.querySelector('input[value="Continue"]');
+  const continueButton = document.querySelector('button[type="submit"]');
   const checkbox = document.querySelector('input[name="ial2_consent_given"]');
 
   function sync() {
-    continueButton.disabled = !checkbox.checked;
-    continueButton.classList.toggle('btn-disabled', continueButton.disabled);
+    continueButton.classList.toggle('btn-disabled');
```

Comment:
This was changed to sync to the current value of the checkbox in #4157 to prevent issues where the button could become (now appear as) disabled if the user clicked fast enough before JavaScript has a chance to load.

Based on my understanding of what this pull request is doing, I see no reason it needs to change.

```suggestion
    continueButton.classList.toggle('btn-disabled', !checkbox.checked);
```